docs/field-configurations.md

```diff
@@ -0,0 +1,21 @@
+# Field generation configurations
+
+Field generation configurations are applied to fields defined in Fields definition file to tweak how the data is generated.
+
+They must be added to a file named `configs.yml` in the assets template folder of a data stream.
+
+## Config entries definition
+
+The config file is a yaml file consisting of an array of config entry.
+
+For each config entry the following fields are available:
+- `name` *mandatory*: dotted path field, as in `fields.yml`
+- `fuzziness` *optional (`long` and `double` type only)*: delta from the previous generated value for the same field
```

Comment:
let's remember to replicate here the information from https://github.com/elastic/elastic-integration-corpus-generator-tool/pull/46/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R257-R259

Reply:
I'll wait for #46 to be closed before moving forward.